Configure

```diff
@@ -919,8 +919,17 @@ $target{dso_extension}=$target{shared_extension_simple};
 $config{cross_compile_prefix} = $ENV{'CROSS_COMPILE'}
     if $config{cross_compile_prefix} eq "";
 
+# This is textbook from the perlvar manual.  It makes sure we have
+# a perl path that can be used as a file name, or as the case may be,
+# as something that can be used with the hashbang script syntax.
+use Config;
+my $secure_perl_path = $Config{perlpath};
```

Comment:
Yes, I suggest less paranoia. As for #1202 obsoleting this. Maybe not, maybe check for $Config{_exe} makes sense for perl invocations performed during actual build procedure. On the other hand we probably should have noticed such cases by now. In such case, yes, #1202 would obsolete this...